### PR TITLE
Update the Solana Devnet guide and feeds

### PR DESCRIPTION
**Pull request by dwightjl**
Supporting code samples available in https://github.com/smartcontractkit/solana-starter-kit

Vercel preview URL: https://documentation-git-fork-dwightjl-dwightjl-solana-chainlinklabs.vercel.app

### CODE REVIEW

File: docs/Solana/DataFeeds/using-data-feeds-solana.md [hunk 1]
```diff
@@ -11,214 +11,180 @@ metadata:
...
-    If the script executes correctly, you will see output with the current price of SOL/USD.
+    To get prices for a different asset, change `const CHAINLINK_FEED` to a different feed address.
+    <!-- TODO: Change this to a command line argument after the capability is added in client.js. -->
 
-## Review the example code
+1. Each request costs an amount of SOL that is subtracted from the `temporary-id.json` wallet. Run `solana balance` to check the remaining balance for your temporary wallet on Devnet.
 
-You can view the Rust code and Typescript for this example on GitHub. See the [chainlink-solana-demo](https://github.com/smartcontractkit/chainlink-solana-demo) repository. The example code has a few main components:
+  ```bash
+  --keypair ./temporary-id.json --url devnet
```
Comment (archseer) [inline docs/Solana/DataFeeds/using-data-feeds-solana.md:131]:
```suggestion
solana balance --keypair ./temporary-id.json --url devnet
```

File: docs/Solana/DataFeeds/using-data-feeds-solana.md [hunk 2]
```diff
@@ -11,214 +11,169 @@ metadata:
...
 
-    ```bash
-    yarn
-    ```
+Use your deployed program to retrieve price data from a Chainlink data feed on Solana Devnet. For this example, call your deployed program using the [Anchor Web3 module](https://project-serum.github.io/anchor/ts/modules/web3.html) and the [`client.js` example](https://github.com/smartcontractkit/chainlink-solana/blob/develop/contracts/examples/hello-world/client.js) code.
 
-1. Run `yarn start` to execute the script:
+1. In the `hello-world` directory, edit the `client.js` file and replace `<YOUR-PROGRAM-ID>` with the ID for the program that you just deployed.
+
+1. Run the `client.js` example with the `ANCHOR_PROVIDER_URL` and `ANCHOR_PROVIDER_URL` parameters:
```
[the remark this answers is not in the document]
Comment (dwightjl) [reply docs/Solana/DataFeeds/using-data-feeds-solana.md]: Fixed in latest commit.

File: docs/Solana/DataFeeds/using-data-feeds-solana.md (hunk 1 above)
Comment (dwightjl) [reply docs/Solana/DataFeeds/using-data-feeds-solana.md]: Fixed in latest commit.

File: docs/Solana/DataFeeds/using-data-feeds-solana.md [hunk 3]
```diff
@@ -11,214 +11,169 @@ metadata:
...
-        let feed_account = next_account_info(accounts_iter)?;
+1. Run `solana program close` and specify the program that you want to close. The remaining SOL balance returns to your wallet:
 
-        const DECIMALS: u32 = 9;
+    ```bash
+    solana program close [YOUR_PROGRAM_ID] --keypair ./temporary-id.json --url devnet
+    ```
 
-        let price = chainlink::get_price(&chainlink::id(), feed_account)?;
+1. If you have deployments that failed, they might still be in the buffer holding SOL tokens. Run `solana program show` again with the `--buffers` flag:
```
[the remark this answers is not in the document]
Comment (dwightjl) [reply docs/Solana/DataFeeds/using-data-feeds-solana.md:157]: Awesome!

**Review by ritika-b — APPROVED** [review]

**Review by pappas999 — APPROVED** [review]
LGTM

**Comment by dwightjl** [comment]
Force pushed to restore auto-generation on data feeds lists

**Pull request merged**